[TA] Add known issues (#19272)
Azure/azure-sdk-for-net: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

### Fixes
- `RecognizePiiEntities` and `TextAnalyticsActions.RecognizePiiEntitiesOptions` were always passing `PiiEntityDomainType.PHI`. Now, it is only passed when requested by the user [19086](https://github.com/Azure/azure-sdk-for-net/issues/19086).

### Known Issues

> - The parameter `CategoriesFilter` in `RecognizePiiEntitiesOptions` doesn't have an impact when using it in `StartAnalyzeBatchActions`. [19237](https://github.com/Azure/azure-sdk-for-net/issues/19237).

Reviewer: NIT:
```suggestion
- The parameter `CategoriesFilter` in `RecognizePiiEntitiesOptions` is currently not working when used in `StartAnalyzeBatchActions`. [19237](https://github.com/Azure/azure-sdk-for-net/issues/19237).
```
